### PR TITLE
function to return model compounds

### PR DESCRIPTION
**Pull request by els1**
Added function to helpers.py to return the model compounds: total continuum, total dust continuum, total dust features, total H2 lines.

### CODE REVIEW

**Comment by els1** [comment]
Issue #19: IDL version gives (in addition to individual components)

TOTAL_DUST_CONTINUUM=dc_tot, TOTAL_CONTINUUM=tot_cont,                         TOTAL_DUST_FEATURES=df_tot, LINES=lines,TOTAL_LINES=lines_tot, $
EXTINCTION_CURVE=ext_curve

I didn't add:
total lines, only total H2 lines
extinction curve

Should this be added?

**Comment by karllark** [comment]
The codacy failure is minor and is due to trailing white space on some lines.  This is a PEP8 thing.  It can be left or fixed if you want.  Click on the Details link and and then the view more details to see the specific lines that have this issue.

**Comment by karllark** [comment]
I think you should add a compound that is the atomic emission lines at a minimum (currently this code returns the H2 only).  

I would return the attenuation curve as well.  Useful as in the future we will have a more complex attenuation curve (e.g., ices with the v2.1 AKARI version).

**Comment by els1** [comment]
> The codacy failure is minor and is due to trailing white space on some lines. This is a PEP8 thing. It can be left or fixed if you want. Click on the Details link and and then the view more details to see the specific lines that have this issue.

Thanks. I think I removed them.

**Review by karllark — APPROVED** [review]
Looks good to me.

**Comment by karllark** [comment]
One thought.  It would be good to have some documentation on this useful function.  Like a doc page that shows the output for our standard M101 nuclear fit.  This could be part of this PR.  But doesn't have to be.  If not, then an issue should be opened so that such docs are added at some point.

**Comment by els1** [comment]
> One thought. It would be good to have some documentation on this useful function. Like a doc page that shows the output for our standard M101 nuclear fit. This could be part of this PR. But doesn't have to be. If not, then an issue should be opened so that such docs are added at some point.

I will work on including the docs in this PR.

File: pahfit/helpers.py [hunk 1]
```diff
@@ -8,6 +8,11 @@
 
 from pahfit.base import PAHFITBase
 
+from pahfit.component_models import BlackBody1D, S07_attenuation
+from astropy.modeling.physical_models import Drude1D
+from astropy.modeling.functional_models import Gaussian1D
+
+
 __all__ = ["read_spectrum", "initialize_model", "fit_spectrum"]
```
Comment (ThomasSYLai) [inline pahfit/helpers.py:16]: add "calculate_compounds" in __all__

**Review by ThomasSYLai — CHANGES_REQUESTED** [review]
Everything looks good to me except adding "calculate_compounds" in __all__

File: pahfit/helpers.py (hunk 1 above)
Comment (els1) [reply pahfit/helpers.py]: corrected. thanks.

**Review by ThomasSYLai — APPROVED** [review]

**Comment by els1** [comment]
I will add the docs in a new PR as there is an active PR on docs.

**Comment by karllark** [comment]
So, is this PR ready for merging @els1?

**Comment by els1** [comment]
@karllark yes it is.

**Pull request merged**